wagtail/admin/templates/wagtailadmin/shared/side_panels/preview.html

```diff
@@ -1,64 +1,100 @@
+            </label>
+            <label class="w-preview__size-button">
+                {% icon name="desktop" %}
+                <input type="radio" name="preview-size" value="desktop" data-action="w-preview#togglePreviewSize" data-w-preview-target="size" data-device-width="1280" aria-label="{% trans 'Preview in desktop size' %}" />
+            </label>
+        {% endblock %}
 
-<div class="preview-panel preview-panel--mobile" data-preview-panel data-action="{{ preview_url }}">
-    <div class="preview-panel__sizes">
-        <label class="preview-panel__size-button preview-panel__size-button--mobile">
-            {% icon name="mobile-alt" %}
-            <input type="radio" name="preview-size" value="mobile" data-device-width="375" data-default-size checked aria-label="{% trans 'Preview in mobile size' %}" />
-        </label>
-        <label class="preview-panel__size-button preview-panel__size-button--tablet">
-            {% icon name="tablet-alt" %}
-            <input type="radio" name="preview-size" value="tablet" data-device-width="768" aria-label="{% trans 'Preview in tablet size' %}" />
-        </label>
-        <label class="preview-panel__size-button preview-panel__size-button--desktop">
-            {% icon name="desktop" %}
-            <input type="radio" name="preview-size" value="desktop" data-device-width="1280" aria-label="{% trans 'Preview in desktop size' %}" />
-        </label>
-
-        <a href="{{ preview_url }}?mode={{ object.default_preview_mode|urlencode }}" target="_blank" rel="noreferrer" class="preview-panel__size-button preview-panel__size-button--new-tab" data-preview-new-tab aria-label="{% trans 'Preview in new tab' %}">{% icon name="link-external" %}</a>
-    </div>
-
-    <div class="preview-panel__spinner w-hidden" data-preview-spinner>
-        {% icon name="spinner" classname="default" %}
-        <span class="w-sr-only">{% trans 'Loading'%}</span>
+        {% block new_tab_button %}
+            <a href="{{ preview_url }}?mode={{ object.default_preview_mode|urlencode }}" target="_blank" rel="noreferrer" class="w-preview__size-button" data-w-preview-target="newTab" data-action="w-preview#openPreviewInNewTab" aria-label="{% trans 'Preview in new tab' %}">{% icon name="link-external" %}</a>
+        {% endblock %}
     </div>
 
-    {% if not settings.WAGTAIL_AUTO_UPDATE_PREVIEW %}
-        <button class="button button-small button-secondary button--icon preview-panel__refresh-button" data-refresh-preview>{% icon name="rotate" %} Refresh</button>
+    {% if auto_update_interval %}
+        <div class="w-preview__spinner" data-w-preview-target="spinner" hidden>
+            {% icon name="spinner" classname="default" %}
+            <span class="w-sr-only">{% trans 'Loading'%}</span>
+        </div>
```

Comment:
This spinner is not used if the auto-update is disabled, in favour of using a w-progress button on the refresh button itself.